optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ExamplesOverview/ExamplesOverview-section.adoc

```diff
@@ -3,23 +3,23 @@
 
 Planner has several examples. In this manual we explain mainly using the _n_ queens example and cloud balancing example. So it is advisable to read at least those sections.
 
-The `Competition?` column in the following table identifies an example as being either realistic or unrealistic. A _realistic competition_ is _an official, independent competition_:
+Some of the examples solve problems that are presented in academic contests. The `Contest` column in the following table lists the contests. It also identifies an example as being either _realistic_ or _unrealistic_ for the putpose of a contest. A _realistic contest_ is _an official, independent contest_:
```

Comment:
typo: putpose

Let's remove the entire sentence "It also identifies an example as being either _realistic_ or _unrealistic_ for the putpose of a contest." as the definition of "realistic" is immediately after.

Reply:
WIll fix typo but disagree about the sentence. Without this sentence, the definition of "realistic" simply comes out of nowhere. 

The sentence is a stylistic/logical connection between "table lists the contents" and "a realistic contest is...". We must first introduce the fact that a contest can be "realistic" or "unrealistic" and only then define the criteria for that.

Reply:
Ok, although I think "for the purpose of the contest" is dead weight, I don't mind keeping it in. Will merge once typo is fixed.